### PR TITLE
Braian.quinteros/motionlayout

### PR DESCRIPTION
**Pull request by BQuinteros**
Changed the principal UI to a better solution using a motion layout.
ezgif com-video-to-gif (6)

### CODE REVIEW

**Review by SartiMau — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Comment by BQuinteros** [comment]
how it is working the app: 
ezgif com-video-to-gif (7)
How it looks the image view on XML start: 
image
How it looks the image view on XML end: 
image

**Review by SartiMau — COMMENTED** [review]
can you add some styles in the layouts instead of all the properties?

**Review by mauroLonghi — APPROVED** [review]

File: WheaterApp/app/src/main/res/layout/activity_city.xml [hunk 1]
```diff
@@ -1,38 +1,48 @@
...
+        android:id="@+id/city_activity_background"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:src="@drawable/icon"
+        android:scaleType="centerCrop"
+        android:src="@drawable/city_activity_background"
+        app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <TextView
```
Comment (SartiMau) [inline WheaterApp/app/src/main/res/layout/activity_city.xml:19]: background and padding can be in the style, put it there pls

File: WheaterApp/app/src/main/res/layout/activity_city_motion_scene_end.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        android:padding="@dimen/padding_city_activity_ten_dp"
+        android:text="@string/city_activity_search_text"
+        app:layout_constraintBottom_toTopOf="@id/main_edit_text_country" />
+
+    <TextView
+        android:id="@+id/validCityTextView"
+        style="@style/BasicSizeAndColor"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_top_city_activity_one_hundred_dp"
```
Comment (SartiMau) [inline WheaterApp/app/src/main/res/layout/activity_city_motion_scene_end.xml]: this can be inside the style

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: WheaterApp/app/src/main/res/layout/activity_city.xml (hunk 1 above)
Comment (BQuinteros) [reply WheaterApp/app/src/main/res/layout/activity_city.xml:19]: Done

File: WheaterApp/app/src/main/res/layout/activity_city_motion_scene_end.xml (hunk 2 above)
Comment (BQuinteros) [reply WheaterApp/app/src/main/res/layout/activity_city_motion_scene_end.xml]: Done

**Review by nicolbidone — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**